x-pack/docs/en/rest-api/security/get-roles.asciidoc

```diff
@@ -24,7 +24,9 @@ privilege.
 ==== {api-description-title}
 
 For more information about the native realm, see 
```

Comment:
You could optionally add the same introductory sentence here:
```suggestion
The roles APIs are generally the preferred way to manage roles, rather than using
{stack-ov}/defining-roles.html#roles-management-file[file-based role management]. 
The get roles API cannot retrieve roles that are defined in roles files.

For more information about the native realm, see 
```